airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py

```diff
@@ -98,6 +110,17 @@ def test_read(
         records = [message.record for message in output if message.type == Type.RECORD]
         counter = Counter(record.stream for record in records)
 
+        if inputs.validate_schema:
+            streams_with_errors = set()
+            for record, errors in verify_records_schema(records, configured_catalog):
+                if record.stream in streams_with_errors:
+                    continue
+                logging.error(f"The {record.stream} stream has the following schema errors: {errors}")
+                streams_with_errors.add(record.stream)
```

Comment:
This seem to be a bit more clear
```suggestion
                if record.stream not in streams_with_errors:
                    logging.error(f"The {record.stream} stream has the following schema errors: {errors}")
                    streams_with_errors.add(record.stream)
```